EatGoodNaija.Server/Model/DTO/profileDTO/UserProfileUpdateDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Org.BouncyCastle.Asn1.Crmf;
+using System.ComponentModel.DataAnnotations;
+
+namespace EatGoodNaija.Server.Model.DTO.profileDTO
+{
+    public class UserProfileUpdateDTO
```

Comment:
You're repeating code here (violating DRY). You can make this class inherit from UserProfileCreateDTO. E.g

public class UserProfileUpdateDTO : UserProfileCreateDTO
{
}

Reply:
implemented sir